src/OrbitGl/TrackManager.cpp

```diff
@@ -468,6 +475,20 @@ CGroupAndProcessMemoryTrack* TrackManager::CreateAndGetCGroupAndProcessMemoryTra
   return cgroup_and_process_memory_track_.get();
 }
 
+PagefaultTrack* TrackManager::CreateAndGetPagefaultTrack(
```

Comment:
I would prefer to only use Create in the name and still return the track.

So CreatePagefaultTrack()

Reply:
`CreatePagefaultTrack` sounds like we are not returning the track. We have the similar naming for `CreateAndGetConfigPath`.

Reply:
I don't like it. But my personal knowledge about C++ isn't the best :P . Leave it there if you prefer, maybe I will propose a change in the future when I make a PR to split Get and Create methods :)